### ISSUE TITLE
Jaccard computation

### ISSUE THREAD

**Issue by caumente** [rundherum/pymia#49]
I'm encountering some issues with the calculation of the Jaccard metric. It seems that pymia assigns infinity to the Jaccard metric when tp, fp, and fn are all 0. Mathematically, handling the division by zero makes sense, but from a medical imaging perspective, it’s not as clear to me. In fact, this case is well-handled in a similar metric, the Dice coefficient. There, the following control exists:

```python
if (self.confusion_matrix.tp == 0) and ((self.confusion_matrix.tp + self.confusion_matrix.fp + self.confusion_matrix.fn) == 0):
    return 1.
```

This makes sense, given that if both the segmentation and the prediction are empty, the Dice should be 1, not infinity. Should't be the same for Jaccard?

**Comment by fabianbalsiger** [comment]
Hi, yes, you are right, it should be handled as in the `DiceCoefficient`. How about you create a pull request with the modification? I am happy to merge and make a new release.

**Comment by caumente** [comment]
Great @fabianbalsiger! Thanks for your prompt answer.

**Comment by fabianbalsiger** [comment]
I just released v0.3.3 including this fix. Thanks for your contribution!
